packages/a2ui/src/instantiate.ts

```
@@ -0,0 +1,195 @@
+
+/**
+ * Write direction: local, immediate, no network. We resolve WHERE the write
+ * lands; the catalog entry decides WHICH handler prop carries it, because
+ * every input names its change event differently (`onInput`,
+ * `onCheckedChange`, `onValueChange`).
+ */
+function writeFor(
+  definition: ComponentDefinition,
+  ictx: InstantiateContext,
+  scope: Scope,
+): WriteFn | null {
```

ESLint

[error] Replace `⏎··definition:·ComponentDefinition,⏎··ictx:·InstantiateContext,⏎··scope:·Scope,⏎` with `definition:·ComponentDefinition,·ictx:·InstantiateContext,·scope:·Scope`

(prettier/prettier)